.run/Cosv-Frontend-LocalRun.run.xml

```diff
@@ -0,0 +1,23 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
where do you put resources (static) like images and gifs for COSV?

Reply:
they should also be split

Reply:
Removed unnecessary images from _cosv-frontend_